### ISSUE TITLE
Should not trigger on braces in when statement 

### ISSUE THREAD

**Issue by aktsay6** [cqfn/diKTat#787]
## Describe the bug
Now it is triggering on braces in `when` statement, when there is a comment inside  
## Expected behavior
```kotlin
when(smth) {
  is String -> {
        // some comment
        code
  }
}
```
## Observed behavior
```kotlin
when(smth) {
  // some comment
  is String -> code
}
```

## Environment information
* diktat version: 0.4.1

**Comment by petertrr** [comment]
Maybe it was originally intended behavior? Do we have any tests on similar cases in this braces rule? @aktsay6 could you please check?

**Comment by aktsay6** [comment]
There are no tests on this behavior

**Comment by petertrr** [comment]
> There are no tests on this behavior

https://github.com/cqfn/diKTat/blob/master/diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/BracesRuleWarnTest.kt#L132

**Comment by aktsay6** [comment]
Missed it. Guess we can close this issue
